build/build.xml

```diff
@@ -72,12 +72,31 @@
         <pathelement location="${root}/reladomo/target/reladomo-asm-4.2.jar"/>
     </path>
 
+    <path id="reladomographql.compile.classpath">
+        <pathelement location="${root}/reladomo/target/classes"/>
+        <pathelement location="${root}/build/lib/graphql/graphql-java-13.0.jar"/>
+        <pathelement location="${root}/build/lib/graphql/graphql-java-servlet-8.0.0.jar"/>
+        <pathelement location="${root}/lib/serial/jackson-databind-2.8.6.jar"/>
+        <pathelement location="${root}/lib/serial/jackson-core-2.8.6.jar"/>
+        <pathelement location="${root}/build/lib/graphql/javax.servlet-api-3.0.1.jar"/>
+        <pathelement location="${root}/build/lib/graphql/guava-24.1.1-jre.jar"/>

```

Comment:
why is this in the compile path?

Reply:
metaSdl = Resources.toString(metaUrl, Charsets.UTF_8);

in the SchemaProvider. If there is a significant concern, I can write it in a few lines myself with ClassLoader.

Reply:
please do.

Reply:
removed guava